build.assets/Dockerfile

```diff
@@ -79,11 +80,25 @@ RUN mkdir -p /opt && cd /opt && curl https://storage.googleapis.com/golang/$RUNT
 
 # Install libbpf
 ARG LIBBPF_VERSION
-RUN mkdir -p /opt && cd /opt && curl -L https://github.com/libbpf/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz | tar xz && \
+RUN mkdir -p /opt && cd /opt && curl -L https://github.com/gravitational/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz | tar xz && \
```

Comment:
A couple questions about this fork:
- why is it forked from `https://github.com/eyakubovich/libbpf` and not `https://github.com/libbpf/libbpf`?
- how are we going to keep the fork up to date with upstream?

Reply:
Will talk with @eyakubovich tomorrow about this and the possibility of upstreaming this change.

Reply:
Please document somewhere why we use a fork and our plans to upstream the patches.